sklearn/tests/test_metaestimators.py

```diff
@@ -120,15 +120,24 @@ def score(self, X, *args, **kwargs):
                         msg="%s does not have method %r when its delegate does"
                             % (delegator_data.name, method))
             # delegation before fit raises a NotFittedError
-            assert_raises(NotFittedError, getattr(delegator, method),
-                          delegator_data.fit_args[0])
+            if method.startswith('score'):
```

Comment:
just `method == 'score'` should work?

Reply:
Yes, sorry.

Reply:
And I saw that the other day when skimming through, but missed it today!